### PR TITLE
Started Windows Label, TextField & Button

### PR DESCRIPTION
**Pull request by Salakar**
Styles still need some changes, but the base is there.

Also think maybe we should have a component loader/bootstrap of sorts so that we don't have to repeat all that code for a common component?

### CODE REVIEW

File: src/Button.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import { OS } from './Desktop';
+import ButtonOSX from './Button/PushButton.osx';
+import ButtonWindows from './Button/Button.windows';
+
+class Button extends Component {
+  static propTypes = {
+    style: PropTypes.object,
+    form: PropTypes.any,
+    visible: PropTypes.bool,
+    display: PropTypes.bool
```
Comment (gabrielbull) [inline src/Button.js]: We'd need to have the same propTypes as the child buttons:

```js
{
  children: PropTypes.oneOfType([React.PropTypes.string, React.PropTypes.element, React.PropTypes.array]).isRequired,
  form: PropTypes.any,
  color: PropTypes.string,
  style: PropTypes.object,
  onClick: PropTypes.func,
  onPress: PropTypes.oneOfType([React.PropTypes.string, React.PropTypes.func]),
  visible: PropTypes.bool,
  display: PropTypes.bool
}
```

**Comment by gabrielbull** [comment]
Oops I think the unit test is failing which prevents me from merging. 

>Also think maybe we should have a component loader/bootstrap of sorts so that we don't have to repeat all that code for a common component?

What about the propsTypes? I think people are using them to know what props a component can receive.

**Comment by Salakar** [comment]
Aye its because I renamed PushButton to Button, forgot to update the tests so its still looking for the old one

**Comment by Salakar** [comment]
There we go, fixed

**Comment by Salakar** [comment]
There we go, fixed the conflicts also, ready to pull! The windows styles won't show on the playground btw as 

```js
Desktop.OS = 'win';
```
will no longer work because of that other pull request by @chentsulin made it a const.

**Pull request merged**